### DISCUSSION TITLE
Improve "Ensure that bump command exists"

### DISCUSSION THREAD

**Discussion by dsherret** [swc-project/swc#3702] [General]
It would be good to improve this message. As a contributor to swc, I see the CI fails and then have to inspect the code of ensure-comment.ts wondering why it's failing. Maybe it could say there's nothing actionable the contributor needs to do (or ideally, I wonder if there's a way that this could be a pending status check until that comment occurs... I'm not familiar enough with GH actions)

![image](https://user-images.githubusercontent.com/1609021/155357691-695f3b0c-c90c-4a7d-965a-43c0eb4911e9.png)

**Comment by kdy1** [comment]
I want to make it pending, but I'm not sure the way to do it.
Btw, I agree that the error message should be improved

**Comment by kdy1** [comment]
I patched it to be `Expected` state before review.
This will not be rendered as an error by default. (If someone other than me reviews it, it will result in an error, though)

**Reply by kdy1** [reply]
I reverted the patch, because it requires a review comment after auto-rebase.
